docs/installation.md

```diff
@@ -7,77 +7,69 @@ and provide feedback. We do not provide support for day-to-day usage of Metals.
+- (sbt 1) `~/.sbt/1.0/plugins/plugins.sbt`
+- (sbt 0.13) `~/.sbt/0.13/plugins/plugins.sbt`
 
-From the repo root run `sbt publishLocal`
+## VSCode extension
+The VSCode extension is [published on the Marketplace](https://marketplace.visualstudio.com/items?itemName=scalameta.metals).
+You can open VSCode and search for it or click [here](vscode:extension/scalameta.metals) to install
+it directly.
 
 # Per-project setup
 
 These steps are required on each project.
 
-## Step 1 - add semanticdb-scalac compiler plugin to your project
+## Quick-start
+The quickest way to get started with Metals is to use the `metalsSetup` command in sbt.
 
-Some features like definition/references/hover rely on artifacts produced by a
-compiler plugin called `semanticdb-scalac`. There are two alternative ways to
-install `semanticdb-scalac`.
-
-The first option is to enable `semanticdb-scalac` permanently for your project
-in `build.sbt` with:
-
-```scala
- addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.8" cross CrossVersion.full)
-scalacOptions += "-Yrangepos"
+```
+sbt
+> metalsSetup
 ```
 
-The second option is to enable `semanticdb-scalac` only for an active sbt
-session by running `semanticdbEnable` from the sbt shell.
+The command will create the necessary metadata in the `.metals` directory
+(which you should not checkout into version control) and setup the `semanticdb-scalac` compiler
+plugin for the current sbt ession.
 
-```scala
-$ sbt
-> semanticdbEnable // automatically runs libraryDependencies += compilerPlugin(...)
-> compile // re-compile project with semanticdb-scalac compiler plugin
-> ...
-```
+You should not checkout the `.metals` directory into version control. We recommend to add it to your
+project's `.gitignore` or/and to your global `.gitignore`:
 
-As soon as you exit the sbt shell you need to re-run `semanticdbEnable` next
-time you open sbt.
+```
+echo ".metals/" >> .gitignore
+```
 
-## Step 2 - produce the build metadata
+Note that you will need to invoke `metalsSetup` (or `semanticdbEnable`) whenever you close and
+re-open sbt. For a more persistent setup, keep reading.
 
-In your project of choice, open `sbt` and run `metalsSetup`. Running the task
-produces the necessary metadata for the server to support features like
-completions and goto definition in dependency sources.
+## Persisting the semanticdb-scalac compiler plugin
+Some features like definition/references/hover rely on artifacts produced by a compiler plugin
+called `semanticdb-scalac`.
 
-> **NOTE**: you will need to repeat this step every time you add a new
-> dependency in your build or when you run `sbt clean`.
+`metalsSetup` enables the plugin on the current session (by invoking `semanticdbEnable`), but you
+can choose to enable it permanently on your project by adding these two settings in your sbt build
+definition:
 
-## Step 3 - start editing
+```scala
+addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.8" cross CrossVersion.full)
```

Comment:
Let's add

```
val semanticdbScalac = "org.scalameta" % "semanticdb-scalac" % "2.1.8" cross CrossVersion.full
```

to the `autoImport` section of MetalsPlugin and recommend users to install instead

```
addCompilerPlugin(semanticdbScalac)
```

This means when users upgrade metals they automatically upgrade scalameta as well.